### DISCUSSION TITLE
一个想法，做个全平台的 nft 浏览器

### DISCUSSION THREAD

**Discussion by NftTopBest** [rebase-network/hello-world#3] [Ideas]
不知道有没有人已经有开发过这一整套的体系？

1. 跨平台：mobile/各大平台小程序/web/chorome 插件等
2. 方便小白用户查看 NFT
3. 其他功能可以联想的其实非常多。

**Comment by luckyyang** [comment]
有点意思，其它的交易不管吗，只搞 nft 相关的吗

**Reply by NftTopBest** [reply]
初期就搞个核心功能呗，有用户啊，或者发了 NFT 有人买了再加各种功能都是可以的。
目前可以搞的比如去中心化 NFT 交易所啥的。

**Comment by zhangtaotaoBTC** [comment]
想法很好，这个应该就是钱包app 和 NFT交易网站想要做的吧，显示你地址上拥有哪些NFT

**Reply by NftTopBest** [reply]
其实来源于我一个朋友他们公司区块链团队给他发了个 BSC 的 NFT，然后不知道怎么看这个 NFT。。
然后我想如果搞个比如微信小程序里面可以直接输入地址选择链就能直接看NFT 应该会方便一些。当然要受众大得全平台支持咯。

**Reply by NftTopBest** [reply]
然后后面加缓存啥的，想到加缓存，就感觉 NFT 缓存也是可以做成一个底层服务给其他平台调用的。

**Reply by luckyyang** [reply]
感觉这个需求是有的，可以看自己所有平台的 nft
